docs/COMMIT-GUIDELINES.md

```diff
@@ -42,7 +42,7 @@ Must be one of the following:
 - `minifix`: A minimal fix that doesn't warrant an entry in the CHANGELOG.
 - `docs`: Documentation only changes.
 - `style`: Changes that do not affect the meaning of the code (white-space,
-formatting, missing semi-colons, etc).
+formatting, missing semi-colons, annotations, etc).
```

Comment:
Sorry if it's a silly question, but what does annotations mean? Regular comments and JSDoc?